### ISSUE TITLE
TypeError at /spi/ip-analysis/Chowkatsun9/

### ISSUE THREAD

**Issue by roysmith** [roysmith/spi-tools#68]
```
Environment:


Request Method: GET
Request URL: https://spi-tools-dev.toolforge.org/spi/ip-analysis/Chowkatsun9/

Django Version: 2.2.13
Python Version: 3.7.3
Installed Applications:
['django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'cat_checker',
 'spi',
 'pageutils',
 'tools_app.apps.ToolsAppConfig',
 'tools_app.templatetags.tools_tags',
 'social_django',
 'debug_toolbar']
Installed Middleware:
['debug_toolbar.middleware.DebugToolbarMiddleware',
 'django.middleware.security.SecurityMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware',
 'social_django.middleware.SocialAuthExceptionMiddleware',
 'tools_app.middleware.RequestAugmentationMiddleware',
 'tools_app.middleware.LoggingMiddleware']



Traceback:

File "/data/project/spi-tools-dev/www/python/venv/lib/python3.7/site-packages/django/core/handlers/exception.py" in inner
  34.             response = get_response(request)

File "/data/project/spi-tools-dev/www/python/venv/lib/python3.7/site-packages/django/core/handlers/base.py" in _get_response
  115.                 response = self.process_exception_by_middleware(e, request)

File "/data/project/spi-tools-dev/www/python/venv/lib/python3.7/site-packages/django/core/handlers/base.py" in _get_response
  113.                 response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/data/project/spi-tools-dev/www/python/venv/lib/python3.7/site-packages/django/views/generic/base.py" in view
  71.             return self.dispatch(request, *args, **kwargs)

File "/data/project/spi-tools-dev/www/python/venv/lib/python3.7/site-packages/django/views/generic/base.py" in dispatch
  97.         return handler(request, *args, **kwargs)

File "./spi/views.py" in get
  125.         summaries.sort()

Exception Type: TypeError at /spi/ip-analysis/Chowkatsun9/
Exception Value: '<' not supported between instances of 'IpSummary' and 'IpSummary'

```

**Comment by roysmith** [comment]
Also: TypeError at /spi/ip-analysis/Kabylewarrior15/

**Comment by roysmith** [comment]
Fixed in 22b09a78fec3939f82ed20ed5a0279e42a3a3dcf
